bundle/config/resources/volume.go

```diff
@@ -33,8 +36,24 @@ func (v Volume) MarshalJSON() ([]byte, error) {
 	return marshal.Marshal(v)
 }
 
-func (v *Volume) Exists(ctx context.Context, w *databricks.WorkspaceClient, id string) (bool, error) {
-	return false, errors.New("volume.Exists() is not supported")
+func (v *Volume) Exists(ctx context.Context, w *databricks.WorkspaceClient, fullyQualifiedName string) (bool, error) {
+	_, err := w.Volumes.Read(ctx, catalog.ReadVolumeRequest{
+		Name: fullyQualifiedName,
+	})
+	if err != nil {
+		log.Debugf(ctx, "volume with fully qualified name %s does not exist: %v", fullyQualifiedName, err)
+
+		var aerr *apierr.APIError
+		if errors.As(err, &aerr) {
+			if aerr.StatusCode == 404 {
+				return false, nil
+			}
+		}
```

Comment:
You can use `apierr.IsMissing(err)` as shorthand for this.

This, in turn, is a shorthand for `errors.Is(err, apierr.ErrNotFound)`.

Reply:
Changed in #2619